### PR TITLE
Correct misalignment between challenge names and hour estimates

### LINKED ISSUE #15706: Hour estimates don't wrap properly
The hour estimates (for example: "(5 hours)") will get pushed down to the subsequent line if the map viewport gets narrow enough, and they no longer line up with the challenges they describe.

jquery__target_html_elements_with_selectors_using_jquery___freecodecamp

We need someone to dig in and figure out a way these can float right without getting misaligned with subsequent lines.

### PR DESCRIPTION
**Pull request by anawebdev**
#### Pre-Submission Checklist

- [X] Your pull request targets the `staging` branch of freeCodeCamp.
- [X] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [X] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [X] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [X] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [X] Tested changes locally.
- [X] Closes currently open issue (replace XXXX with an issue no): Closes #15706

#### Description

The misalignment happens when the left panel in challenges view gets too small and the text wraps around the hour estimates, which are floated right. I added two media queries to fix this.

### CODE REVIEW

File: common/app/Map/map.less [hunk 1]
```diff
@@ -88,6 +88,18 @@
...
 }
 
+.@{ns}-accordion-panel-title {
+  @media (min-width: 721px) {
+    clear: both;
+  }
+}
+
+.@{ns}-challenge-title {
+  @media (min-width: 721px) {
```
Comment (dhcodes) [inline common/app/Map/map.less]: Could these `@media` lines be added to the selector at line 72?

**Review by dhcodes — CHANGES_REQUESTED** [review]

File: common/app/Map/map.less (hunk 1 above)
Comment (anawebdev) [inline common/app/Map/map.less]: Sure :)

**Comment by anawebdev** [comment]
I  made the requested changes.

**Comment by QuincyLarson** [comment]
@anawebdev Awesome. Thanks for this elegant fix!

@dhcodes this looks good to me. Thanks for reviewing this. Please merge once you're ready.

**Review by dhcodes — APPROVED** [review]

**Comment by dhcodes** [comment]
Agree with Quincy. **LGTM**. Thanks @anawebdev for making this change. 👍 💯 🏎 🌮

**Pull request merged**